Pop-Grupa-A-Frontend/src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
+import ComputationCockpit from './components/computationCockpit/ComputationCockpit.js';
```

Comment:
unikałbym tworzenia folderu na pojedynczy komponent.
W przyszłości jak będzie więcej komponentów to może się potworzy jakieś folder związany domenowo z kilkoma komponentami